listenbrainz/webserver/static/js/src/recent/RecentListens.tsx

```diff
@@ -66,13 +64,6 @@ export default class RecentListens extends React.Component<
     this.setState({ recordingToReview });
   };
 
-  getFeedbackForRecordingMsid = (
-    recordingMsid?: string | null
-  ): ListenFeedBack => {
-    const { recordingFeedbackMap } = this.state;
-    return recordingMsid ? get(recordingFeedbackMap, recordingMsid, 0) : 0;
-  };
-
```

Comment:
This page currently does not have feedback anyway because these functions are isolated so feedback doesn't work. It probably makes sense to add it here as well but not sure if in this or other PR.

Reply:
Let's do separate PR, it's not the most important page to heart things on IMO.

Reply:
While testing found that feedback on player page and playlists page is also not working.